backend/src/controllers/muralcollection.controller.ts

```diff
@@ -10,18 +10,23 @@ export class MuralCollectionController {
   public collectionService: MuralCollectionService = new MuralCollectionService();
 
   // POST /collection
+  /**
+   * POST /collection
+   * @param req HTTP request containing a "collection" parameter that has a
+   * MuralCollectionInterface describing the collection, and a list of mural IDs
+   * @param res Http response
```

Comment:
have you been describing the responses as well?

Reply:
Yeah I should, will probably do that in the next pass lol